src/actions/profile-view.js

```diff
@@ -1357,12 +1357,15 @@ export function changeProfileName(
   profileName: string | null
 ): ThunkAction<Promise<void>> {
   return async (dispatch, getState) => {
-    if (window.indexedDB) {
+    if (window.indexedDB && profileName) {
```

Comment:
If profileName is null or empty, we should still store a blank string, otherwise we won't be able to remove it.